src/manifest/manifest.go

```diff
@@ -16,7 +16,6 @@ const (
 	FileName    = "manifest.json"
 	MetaFile    = "meta.json"
 	ConfigFile  = "config.json"
-	RowsDir     = "rows"
```

Comment:
Odstranil som tuto konstantu -> potrebujem aby to bolo v sablone config row `path`, ... vid dalsi comment, ... kedze to bolo riesene inak ako zvysok kodu, tak mi to komplikovalo pridanie novych features.